llama_index/embeddings/base.py

```diff
@@ -217,11 +226,6 @@ async def aget_queued_text_embeddings(
                     self._aget_text_embeddings(cur_batch_texts)
                 )
                 result_ids.extend(cur_batch_ids)
-                self.callback_manager.on_event_end(
```

Comment:
I'm unclear on whether this hook needs to be on a per-batch basis. I moved it to the end of the function after the coroutines had been gathered so I could get access to the computed embeddings.

Reply:
I think it's fine to have it at the end of the function, as long as the non-async version has the same pattern 👍🏻